airflow-core/src/airflow/ui/src/i18n/locales/ar/dashboard.json

```
@@ -0,0 +1,38 @@
+{
+  "group": "المجموعة",
+  "health": {
+    "dagProcessor": "معالج الDag",
+    "health": "الصحة",
+    "healthy": "سليم",
+    "lastHeartbeat": "آخر نبضة",
+    "metaDatabase": "قاعدة البيانات الوصفية",
+    "scheduler": "المجدول",
```

```suggestion
    "scheduler": "المُجَدْوِل",
```
otherwise it could be read as `scheduled`